maintainers/maintainer-list.nix

```diff
@@ -5866,6 +5866,12 @@
     githubId = 22836301;
     name = "Mateusz Mazur";
   };
+  mbaeten = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add mbaeten `` and all the package related changes into a commit with the message ``pragha: init at 1.3.4``.